tests/unit/doubles/actions/importing/abstract-aioseo-settings-importing-action-double.php

```diff
@@ -27,11 +29,27 @@ abstract class Abstract_Aioseo_Settings_Importing_Action_Double extends Abstract
 	 */
 	protected $replacevar_handler;
 
+	/**
+	 * The robots provider service.
+	 *
+	 * @var Aioseo_Robots_Provider_Service
+	 */
+	protected $robots_provider;
+
+	/**
+	 * The robots transformer service.
+	 *
+	 * @var Aioseo_Robots_Transformer_Service
+	 */
+	protected $robots_transformer;
+
 	public function __construct(
 		Options_Helper $options,
-		Aioseo_Replacevar_Handler $replacevar_handler
+		Aioseo_Replacevar_Handler $replacevar_handler,
```

Comment:
You can remove the constructor and all protected properties. The test double only serves a purpose to change visibility of protected methods. The properties and constructor are inherited from `Abstract_Aioseo_Settings_Importing_Action`

Reply:
The same goes for the other test doubles